src/tools/compiletest/src/header.rs

```diff
@@ -723,6 +723,23 @@ impl Config {
         }
     }
 
+    pub fn parse_meta_name_value_directive(&self, line: &str, directive: &str) -> Option<String> {
+        if let Some(eq_index) = line.find("=") {
```

Comment:
I don't think any of the ignore'd tests in src/test/ have a `=` split with some kind of message. Note that this code is *compiletest* which is an internal testing tool for rustc, not the general testing tool used by most Rust users (libtest); the parsing code for that lives in compiler/rustc_builtin_macros/src/test.rs.

Reply:
Thanks for the help, I will figure out the correct implementation in `compiler/rustc_builtin_macros/src/test.rs`

Reply:
The compiletest functionality probably should just be deleted in this PR -- I don't think this function really achieves the desired intent and we can leave it as possible future work (or maybe never).